cern-vdt/00-lsx.patch

```diff
@@ -0,0 +1,29 @@
+--- CMakeLists.txt	2022-03-16 00:11:50.000000000 +0800
++++ CMakeLists2.txt	2024-10-07 10:08:25.146943917 +0800
+@@ -15,12 +15,13 @@
+ option( AVX2 "Use AVX2 instruction set (default cache entry: OFF)" OFF)
+ option( FMA "Use FMA instruction set (default cache entry: OFF)" OFF)
+ option( USERFLAGS "Pass arbitrary flags to the compiler")
+-option( SSE "Use SSE instruction set (default cache entry: ON)" ON)
++option( SSE "Use SSE instruction set (default cache entry: ON)" OFF)
+ option( NEON "Use NEON instruction set (default cache entry: OFF)" OFF)
+ option( BUILD_SHARED_LIBS "Build libraries as SHARED instead of STATIC (default cache entry: ON)" ON)
+ option( PRELOAD "Create in the library the symbols to preload the library (default cache entry: OFF)" OFF)
+ option( USE_VC "Use Vc library - requires symlink to Vc from ${CMAKE_SOURCE_DIR} (default cache entry: OFF)" OFF)
+ option( DEBUG "Compile library with debug symbols (default is OFF)" OFF)
++option( LSX "Use Loongarch LSX instruction set" ON)
```

Comment:
Please refer to upstream's setting:

```cmake
option( NEON "Use NEON instruction set (default cache entry: OFF)" OFF)
```

It seems that they don't enable options other than x86's